packages/flutter/lib/src/material/expand_icon.dart

```diff
@@ -94,21 +95,31 @@ class ExpandIcon extends StatefulWidget {
   /// and for [dark theme](https://material.io/design/color/dark-theme.html#ui-application)
   final Color? expandedColor;
 
+  /// The icon to be displayed.
+  ///
+  /// Defaults to [Icons.expand_more].
+  ///
+  /// This property must not be null.
+  final IconData icon;
+
   @override
   State<ExpandIcon> createState() => _ExpandIconState();
 }
 
-class _ExpandIconState extends State<ExpandIcon> with SingleTickerProviderStateMixin {
+class _ExpandIconState extends State<ExpandIcon>
+    with SingleTickerProviderStateMixin {
   late AnimationController _controller;
   late Animation<double> _iconTurns;
 
-  static final Animatable<double> _iconTurnTween = Tween<double>(begin: 0.0, end: 0.5)
-    .chain(CurveTween(curve: Curves.fastOutSlowIn));
+  static final Animatable<double> _iconTurnTween =
+      Tween<double>(begin: 0.0, end: 0.5)
+          .chain(CurveTween(curve: Curves.fastOutSlowIn));
 
   @override
   void initState() {
     super.initState();
-    _controller = AnimationController(duration: kThemeAnimationDuration, vsync: this);
+    _controller =
```

Comment:
can you revert these format change? in framework we prefer more readability over line limit